content/installation/proxy/ProxyAgentInstall.md

```diff
@@ -39,4 +39,41 @@ After this stage, you're ready to [configure](installation-proxy.html#proxy-conf
 
 The Proxy agent is constructed as a module that plugs into NGINX. It can be compiled statically into NGINX or as an NGINX dynamic module. The software and documentation for building from source is available at: https://github.com/Contrast-Security-OSS/nginx-contrast-connector and https://github.com/Contrast-Security-OSS/nginx-contrast-connector/blob/master/BUILD_FROM_SOURCES.md -->
 
+## Air-Gapped Users
+
+Users that aren't connected to the internet must import Contrast's Linux packages to their internal network since their package manager doesn't reach them. Many organizations have automated import and synchronization procedures for keeping up to date with repositories on the internet disconnected from their air-gapped network. You should ensure your organization doesn't already have an import process that can help with this process. But, regardless of your situation, the following information for manually obtaining the packages is helpful for completing installation.
+
+To manually import the packages, go directly to the location where Contrast hosts packages for your packaging system, and download the newest versions. A dependency listing of what you will need:
+
+	contrast-server -> contrast-modsecurity
+	contrast-webserver-agent-nginx -> nginx
+
+Download the newest version of each of the four packages, and import them to your network. Continue with the distro-specific instructions that follow. After importing and installing the packages, you can move to [configuration](installation-proxy.html#proxy-config). 
```

Comment:
@seschis Does this refer to the download step in the Redhat and Ubuntu instructions that follow?